plugins/DuskVerb/tools/tuner/outdiff_kurtosis_sweep.py

```
@@ -0,0 +1,88 @@
+            if s is None: return 1e3
+            pen += 0.6*max(0.0, s - max(a_sub[b], 5.0))
+        pen += 0.5*max(0.0, a_hf - hf_level(nb[0]) - 1.0)  # don't dull >1 dB below anchor HF
+        return pen
+
+    study = optuna.create_study(direction="minimize", sampler=optuna.samplers.TPESampler(seed=42))
+    optuna.logging.set_verbosity(optuna.logging.WARNING)
+    study.optimize(obj, n_trials=a.trials, n_jobs=a.workers)
+    bp = study.best_params
+    print(f"\nBEST  amount={bp['amount']:.3f} lfoScale={bp['lfoScale']:.3f} delayScale={bp['delayScale']:.3f}  pen={study.best_value:.2f}")
+    # final scoreboard
+    d = "/tmp/odk_best"; shutil.rmtree(d, ignore_errors=True); os.makedirs(d)
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/odk_best"

(S108)